### PR TITLE
Add com.todotxt.sleek

### PR DESCRIPTION
**Pull request by ransome1**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: com.todotxt.sleek.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.todotxt.sleek</id>
+  <name>sleek</name>
+  <summary>todo manager based on the todo.txt syntax for Linux, free and open-source (FOSS)</summary>
+  <launchable type="desktop-id">com.todotxt.sleek.desktop</launchable>
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  <developer_name>Robin Ahle</developer_name>
+  <content_rating type="oars-1.0"/>
```
Comment (bbhtt) [inline com.todotxt.sleek.appdata.xml:10]:
```suggestion
    <content_rating type="oars-1.1"/>
```

File: com.todotxt.sleek.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      <li>Multi line todos can be created</li>
+      <li>Filters are sorted alphanummerically and can be renamed or deleted</li>
+      <li>Hyperlinks detected automatically</li>
+      <li>File watcher scans todo.txt files for changes</li>
+      <li>Simultaneously manage multiple todo.txt files</li>
+      <li>Language options: English, German, Italian, Spanish, French, Simplified Chinese, Brazilian Portugese, Japanese, Turkish, Hungarian, Czech, Polish, Russian</li>
+    </ul>
+  </description>
+  <screenshots>
+    <screenshot type="default">
```
Comment (bbhtt) [inline com.todotxt.sleek.appdata.xml:53]: Pull screenshots from a tag instead of `master`

File: com.todotxt.sleek.appdata.xml
Comment (bbhtt) [inline com.todotxt.sleek.appdata.xml:1]: This should be in the upstream repo

File: com.todotxt.sleek.desktop
Comment (bbhtt) [inline com.todotxt.sleek.desktop:1]: THe deb doesn't have a desktop file? You can use that one

File: com.todotxt.sleek.png
Comment (bbhtt) [inline com.todotxt.sleek.png:1]: THe deb doesn't have an icon? You can use that one

File: generated-sources.json
Comment (bbhtt) [inline generated-sources.json:1]: This doesn't seem to be used in the manifest? Why is it here?

File: com.todotxt.sleek.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=home
+  - --filesystem=/mnt
+  - --filesystem=/media
+  - --filesystem=/run/media
+rename-desktop-file: sleek.desktop
+rename-icon: sleek
+modules:
+  - name: sleek
```
Comment (bbhtt) [inline com.todotxt.sleek.yml:24]: You need a flathub.json if you want to build from x86_64 only

File: com.todotxt.sleek.desktop
Comment (hfiguiere) [reply com.todotxt.sleek.desktop:1]: the file is never referenced. remove it.

File: com.todotxt.sleek.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+          - exec zypak-wrapper /app/sleek/sleek "$@"
+
+    build-commands:
+      - ar x sleek_*.deb
+      - rm -f sleek_*.deb
+      - tar -xf data.tar.xz
+      - rm -f control.tar.gz data.tar.xz debian-binary
+      - cp -r opt/* usr/* $FLATPAK_DEST
+      - rm -rf opt usr
+      - install -Dm644 com.todotxt.sleek.png /app/share/icons/hicolor/128x128/apps/com.todotxt.sleek.png
```
Comment (hfiguiere) [inline com.todotxt.sleek.yml:48]: 512x512 pixel is not 128x128.

Comment (hfiguiere) [reply com.todotxt.sleek.yml:48]: also line 52 seems to indicate there are icons installed, so why here?

**Pull request closed without merging**